lib/doorkeeper/oauth/client_credentials_request.rb

```diff
@@ -12,7 +12,7 @@ class ClientCredentialsRequest
       attr_reader :response
       attr_writer :issuer
 
-      alias :error_response :response
+      alias error_response response
```

Comment:
What do you think about using `alias_method` here instead? See http://stackoverflow.com/a/4763178/356060